swagger.yaml

```diff
@@ -336,3 +422,45 @@ definitions:
           - example:
               id: "E12000001"
               label: "North East"
+  GetDimensionsResponse:
+    description: "The response body containing all available non-geography base variables for a given dataset"
+    properties:
+      area-type:
+        allOf:
+          - $ref: '#/definitions/Dimension'
+          - example:
+              id: "accommodation_type"
+              label: "Accommodation Type"   
+  Dimension:
+    description: "A list of dimensions"
+    type: object
+    properties:
+      count:
+        description: "The number of dimensions returned"
+        readOnly: true
+        type: integer
+      items:
+        type: array
+        items:
+          $ref: '#/definitions/DimensionResponse'
+      limit:
+        description: "The number of dimensions requested"
+        type: integer
+      offset:
+        description: "The first row of dimensions to retrieve, starting at 0. Use this parameter as a pagination mechanism along with the limit parameter"
+        type: integer
+      total_count:
+        description: "The total number of dimensions"
+        readOnly: true
+        type: integer
+  DimensionResponse:
+    description: "The response body containing the information for a dimension"
+    properties:
+      dimension:
+         properties:
+          id:
+            type: string
+          label:
+            type: string
+          total_count:
+            type: integer
```

Comment:
I don't want to be one of "those" people who requires new lines but just an FYI, there's no new line here.

Reply:
New line now added